src/main/java/r4v88/Main.java

```diff
@@ -110,10 +112,13 @@ public static void main(String[] args) throws UserWithIdDoesNotExist, UserWithNa
 //        System.out.println(userService.getAllUsers().toString());
 //        System.out.println(userService.getUserByEmail("imejl.123"));
 
-        BookDao bookDao = BookDaoImpl.getInstance();
-        bookDao.addBook(book);
-//            bookDao.borrowBook(1, false);
-        bookDao.removeBook(1);
+//        BookDao bookDao = BookDaoImpl.getInstance();
+//        bookDao.addBook(book);
+////            bookDao.borrowBook(1, false);
```

Comment:
Zakomentowany kod nie jest nam potrzebny, na gicie masz pełną historię kodu - jak bedzie trzeba do czegos wrocic to za pomocą gita